src/ipv6/ndpv6.ml

```diff
@@ -693,7 +697,7 @@ module RouterList = struct
       end
     | false ->
       if lft > 0L then begin
-	Log.debug (fun f -> f "RA: Adding Router: src=%a" Ipaddr.pp src);
+        Log.debug (fun f -> f "RA: Adding Router: src=%a" Ipaddr.pp src);
```

Comment:
This change is just converting a tab character into spaces. For some reason this completely broke ocp-indent.